pkgs/applications/office/gnote/default.nix

```diff
@@ -0,0 +1,43 @@
+#{ stdenv, lib, meson }:
+with import <nixpkgs> {};
+stdenv.mkDerivation rec {
+  pname = "gnote";
+  version = "42.0";
+
+  src = pkgs.fetchurl {
+    url = "https://gitlab.gnome.org/GNOME/gnote/-/archive/${version}/gnote-${version}.tar.gz";
+    sha256 = "1W1zGNwoM3ea2SCQwC7kpRSUS6n0sAY23Fiffny32k8=";
+  };
+
+  buildInputs = [
+    glib
+    gtkmm3
+    gtk3
+    gspell
+    libxml2
```

Comment:
This should go to nativeBuildInputs if it is only used here:
https://gitlab.gnome.org/GNOME/gnote/-/blob/42.0/data/meson.build#L25